src/data/rat-partner.json

```diff
@@ -1,4 +1,13 @@
 {
+    "section-notice": {
+        "textblock": [
+            "Dear interested persons,",
+            "thank you for your interest in connecting to the Corona-Warn-App. You can find current information about this on <a href='https://github.com/corona-warn-app/cwa-quicktest-onboarding/wiki' target='_blank' rel='noopener noreferrer'>GitHub</a>.",
```

Comment:
"thank you" => "Thank you"

In English correspondence format the first letter of the first word after the salutation is capitalized, unlike in German.

Reply:
Fixed in https://github.com/corona-warn-app/cwa-website/pull/2889/commits/36582ff34e199fc79cddec94613d0c60a30cdd5a, thanks!